### PR TITLE
Port users ldap mail

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: users_ldap_mail/__openerp__.py [hunk 1]
```diff
@@ -21,11 +21,12 @@
 
 {
     'name': "LDAP mapping for user name and e-mail",
-    'version': "1.0",
+    'version': "1.1",
```
Comment (bwrsandman) [inline users_ldap_mail/__openerp__.py]: Why are you bumping the version?
The long version is now 8.0.1.0 which is > 7.0.1.0

Comment (dreispt) [inline users_ldap_mail/__openerp__.py]: Fixed.

**Comment by dreispt** [comment]
Just retested this after https://github.com/odoo/odoo/issues/2146 was fixed, and I confirm it works great!

**Comment by bwrsandman** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Comment by dreispt** [comment]
Could this be merged?

**Comment by dreispt** [comment]
Thanks!

**Pull request merged**